zoria-lang.cabal

```diff
@@ -33,7 +33,10 @@ library
                         Utility,
                         Evaluator,
                         Evaluator.Environment,
-                        Evaluator.ValueSyntax
+                        Evaluator.ValueSyntax,
```

Comment:
In the future prefer to append new modules to the beginning of the list.

Reasoning: since you had to add a comma after ValueSyntax module it appeared in the diff even though it wasn't really modified.